python_utils/ts_log.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+def ts_log(msg):
```

Comment:
why not def ts_log(msg, flush=False)?

Reply:
Because then you'd need to explicitly turn it on every time, or define a new that changes the default to False.
I just figured it makes just as much sense to just give a separate function definition where users otherwise use the exact same code :shrug: